example/assets/config.yml

```diff
@@ -0,0 +1,166 @@
+backend:
+  name: github
+  repo: netlify/netlify-cms
+  branch: cms-test-backend
```

Comment:
This file is used for copy on deploy preview!
The `cms-test-backend` branch is an orphan branch so new changes can be tested on deploy preview.